blog/_posts/2024-08-07-eks-to-aks-series-part1.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Mastering the Move: EKS to AKS by Example"
+description: "Learn how to migrate Amazon EKS workloads to Azure AKS using KEDA and Karpenter in this comprehensive guide."
+date: 2050-08-24
+author: Kenneth Kilty
+categories: general
+---
+
+## Introduction
+
+Many companies use multiple clouds for their workloads. These companies need to accommodate the cloud preferences of their customers. Kubernetes plays a central role in multi-cloud workloads due to its ability to provide a consistent and portable environment across different cloud providers.
+
+We’re excited to announce the first in a new documentation series designed specifically for customers already using Amazon EKS, to help them replicate or migrate their workloads to AKS: [Replicate an AWS event-driven workflow (EDW) workload with KEDA and Karpenter in Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview)
+
+Moving a workload from another managed Kubernetes offering to AKS can be challenging. This series aims to provide a holistic, end-to-end experience by example to help ease that process. The samples in this series begin with infrastructure and code on EKS and end with equivalently functional infrastructure and code on AKS, while explaining the conceptual differences between AWS and Azure throughout.
```

Comment:
Rather than saying that moving a workload from another managed K8s offering can be challenging, I think it would be more accurate to say something like "Even with a portable API like Kubernetes offers, moving between clouds can be challenging". You could also throw in an explanatory metaphor here to make the post a little more informal - eg. "Every cloud has its own unique concepts and behaviors that can seem foreign when you're accustomed to another cloud's approach, not unlike the experience of learning a new language or visiting a new country for the first time. Think of us like your language tutor and local guide to the world of Azure."